src/main/webapp/components/Documents.css

```diff
@@ -1,34 +1,42 @@
-.collaborator-email-container {
-    margin-top: 20px;
+.data-use-clarification {
+    display: block;
+    font-size: 14px;
+    font-weight: 400;
 }
-.collaborator-email-container .inputFieldText {
-    width:calc(100% - 70px);
+
+/* Data Use Letter */
+.yesNo-container .radioOptions  {
     float: left;
 }
 
-.collaborator-email-container .buttonPrimary {
-    float: right;
-    margin: 0;
-    height: 38px;
+.yesNo-container .radioContainer {
+    min-height: 60px;
 }
 
-.shareable-link .btn {
-    height: 38px;
-    font-size: 16px;
-    padding: 6px 30px; 
-    margin: 23px 0 0 90px;
-    float: none;
+.col-padding {
+    padding-right: 30px;
 }
 
-.data-use-clarification {
-    display: block;
-    font-size: 14px;
-    font-weight: 400;
+.checkbox .checkboxLabel {
+    margin-top: 20px;
 }
 
-.collaborator-email-container .errorMessage {
-    display: block;
-    width: 100%;
-    position: absolute;
-    bottom: -62px;
+.datepicker-readonly .checkboxLabel {
+    margin-top: 0;
+}
+
+.readonly-content .react-datepicker__close-icon {
+    display: none;
+}
+
+.alertMessage.alert-success {
+    margin: 20px 0 10px 0;
+}
+
+.getShareable {
+    margin-top: 20px;
+}
+
+.getShareable .glyphicon {
+    margin-right: 5px;
```

Comment:
It would be a lot nicer if we used inline js style objects. Shared css like this is super brittle. I know we've gone down this path for a while, but if possible, we should move in the inline direction.